src/CachePoolTest.php

```diff
@@ -106,6 +106,20 @@ public function testBasicUsage()
         $this->assertFalse($this->cache->getItem('key2')->isHit());
     }
 
+    public function testItemModifiersReturnsStatic()
+    {
+        if (isset($this->skippedTests[__FUNCTION__])) {
+            $this->markTestSkipped($this->skippedTests[__FUNCTION__]);
+
+            return;
+        }
+
+        $item = $this->cache->getItem('key');
+        $this->assertSame($item, $item->set('4711'));
+        $this->assertSame($item, $item->expiresAfter(2));
+        $this->assertSame($item, $item->expiredAt(new \DateTime('+2hours')));
```

Comment:
That should be `expiresAt()`, not `expiredAt()`. Not 100% sure what the value of testing expiry is here - isn't that covered in a different test?